code/__DEFINES/status_effects.dm

```diff
@@ -36,6 +36,7 @@
 
 #define STATUS_EFFECT_GOOD_MUSIC /datum/status_effect/good_music
 
+#define STATUS_EFFECT_LEGION_CORE /datum/status_effect/legion_core
```

Comment:
I would consider changing all these code references to REGENERATIVE_CORE instead of LEGION_CORE, since the legion core is just a child of it with no functional differences.

Reply:
Good call, will do